llvm/lib/DebugInfo/DWARF/DWARFCFIProgram.cpp

```diff
@@ -425,7 +425,7 @@ void CFIProgram::printOperand(raw_ostream &OS, DIDumpOptions DumpOpts,
   case OT_Expression:
     assert(Instr.Expression && "missing DWARFExpression object");
     OS << " ";
-    Instr.Expression->print(OS, DumpOpts, nullptr);
+    DWARFExpressionPrinter::print(&(*Instr.Expression), OS, DumpOpts, nullptr);
```

Comment:
No need for the parents here? Could use .get instead of &* , and wouldn't need the null check in the next case